src/cmd/encoder/eventhandler/consumer.go

```diff
@@ -0,0 +1,97 @@
+package eventhandler
+
+import (
+	contracts "github.com/Sogilis/Voogle/src/pkg/contracts/v1"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/Sogilis/Voogle/src/pkg/clients"
+	"github.com/Sogilis/Voogle/src/pkg/events"
+
+	"github.com/Sogilis/Voogle/src/cmd/api/models"
```

Comment:
I really don't think it's a good idea starting to import each other services.

Reply:
If you speak about "github.com/Sogilis/Voogle/src/cmd/api/models", it will be fix in next PR (#266) where we move video status enum into protobuf contracts